### ISSUE TITLE
using demo code for python for ner outputs logits not actual entities so how to map numbers to entity?

### ISSUE THREAD

**Issue by yash97** [allenai/allennlp#2520]
Please first search our GitHub repository for similar questions.  If you don't find a similar example you can use the following template:

**System (please complete the following information):**
 - OS: [e.g. OSX, Linux]
 - Python version: [e.g. 3.6.1]
 - AllenNLP version: [e.g. v0.8.1, or "I installed from master"]
 - PyTorch version: (if you installed it yourself)

**Question**
Ask about something you don't understand, such as:

* How can I retrain my own ELMo language model?

* I'm working on a sentiment classification task and I would like to discard sentences that are longer than 1200 characters (my sentences vary from 400 characters to 1400+).  How can I configure my model to discard long sentences?

**Comment by matt-gardner** [comment]
Your question is not entirely clear, but I think you're looking for the "tags" output.  If that doesn't help, you're going to need to ask a better question.
